api/logs/serializers.py

```diff
@@ -49,7 +49,6 @@ class NodeLogParamsSerializer(RestrictedDictSerializer):
 
     addon = ser.CharField(read_only=True)
     bucket = ser.CharField(read_only=True)
-    citation_name = ser.CharField(read_only=True, source='citation.name')
```

Comment:
Decided to just remove this while I was in the vicinity the exact same line is at https://github.com/CenterForOpenScience/osf.io/pull/8623/files#diff-b392ea6bde3d90038251a0afc235379cL95